### DISCUSSION TITLE
How to reset default factory Settings for MacOS system?

### DISCUSSION THREAD

**Discussion by alexventuraio** [lapce/lapce#2171] [Q&A] [answered]
I was trying to configure Lapce to my needs and I ended up bloating the UI and I can't find the way to reset them to the default ones.
I have tried uninstalling and reinstalling and it does not work, it keeps loading same settings as before.
Any help?

**Comment by panekj** [comment, marked as answer]
Open palette, type `:setting`, click on `Open Settings Directory` and it will open Lapce data directory that you can delete

**Reply by alexventuraio** [reply]
Thanks, it worked!
I deleted the entire folder and after opening Lapce again, it all was reseted!
